src/System.IO.FileSystem.AccessControl/src/Configurations.props

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
   <PackageConfigurations>
     netstandard;
+    netstandard-Windows_NT;
     netfx-Windows_NT;
     netcoreapp2.0-Windows_NT;
```

Comment:
This configuration becomes redundant I believe.

Reply:
it's not, since we are harvesting `runtimes/win/lib/netstandard1.3/` from previous packages so that will apply to UAP before the netstandard2.0 because it is RID specific. By adding this configuration, then we will use netstandard2.0 impl instead.

Reply:
For this particular case it is still required, the reason is becuase netcoreapp2.0 implementation does have some IL on it, while the netstandard-Windows_NT one just throws PNS so we don't want to have that apply on the netcoreapp2.0 case.